src/views/PurchaseOrderDetail.vue

```diff
@@ -204,6 +253,13 @@ export default defineComponent({
       if(this.queryString) payload = this.queryString
       this.store.dispatch('order/updateProductCount', payload)
     },
+    getPOItemsCount(orderType: string) {
+      if(orderType === 'completed') {
+        return this.order.items.filter((item: any) => item.orderItemStatusId === 'ITEM_COMPLETED').length
+      } else if(orderType === 'pending') {
```

Comment:
```suggestion
      } else {
```

We can remove this `if` as there is no other case we are handling for orderType.